### PR TITLE
CLOUD-2956 Remove JDG stuff from cct_modules

### PR DESCRIPTION
**Pull request by jmtd**
- [x] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

https://issues.jboss.org/browse/CLOUD-2956

Note: JDG 6.* still reference cct_module master, but JDG 6 images
no longer exist in the container catalogue so presumably the
product will never be rebuilt.

### CODE REVIEW

**Comment by osnipso** [comment]
Since Data Grid 7.2, we are not using cct_modules anymore. The last version that did, which was 7.1, is now deprecated, so it's not supposed to be rebuilt. Looks good to me, please merge it. Thank you!

File: tests/features/common.feature [hunk 1]
```diff
@@ -10,30 +10,30 @@ Feature: Openshift common tests
...
-  @jboss-datagrid-6 @jboss-datagrid-7
+  @jboss @jboss
   Scenario: Check that long node names are truncated to 23 characters
     When container is started with env
        | variable  | value                      |
        | NODE_NAME | abcdefghijklmnopqrstuvwxyz |
     Then container log should contain jboss.node.name = defghijklmnopqrstuvwxyz
 
-  @jboss-datagrid-6 @jboss-datagrid-7
+  @jboss @jboss
```
Comment (jmtd) [inline tests/features/common.feature]: bad substitution

File: tests/features/common.feature [hunk 2]
```diff
@@ -10,30 +10,30 @@ Feature: Openshift common tests
...
+  @jboss-kieserver-6 @jboss-decisionserver-6 @jboss-processserver-6 @jboss-webserver-3/webserver30-tomcat7-openshift @jboss-webserver-3/webserver31-tomcat7-openshift @jboss-webserver-3/webserver30-tomcat8-openshift @jboss-webserver-3/webserver31-tomcat8-openshift @jboss-amq-6 @jboss @jboss @jboss-datavirt-6 @redhat-sso-7
   Scenario: Check that script debugging (set -x) can be enabled
     When container is started with env
        | variable     | value |
        | SCRIPT_DEBUG | true  |
     Then container log should contain + echo 'Script debugging is enabled, allowing bash commands and their arguments to be printed as they are executed'
 
   # CLOUD-427: we need to ensure jboss.node.name doesn't go beyond 23 chars
-  @jboss-datagrid-6 @jboss-datagrid-7
+  @jboss @jboss
```
Comment (jmtd) [inline tests/features/common.feature]: bad substitution

File: tests/features/eap/eap_common.feature [hunk 3]
```diff
@@ -216,7 +216,7 @@ Feature: Openshift EAP common tests (EAP and EAP derived images)
      And all files under /opt/eap are writeable by current user
      And all files under /deployments are writeable by current user
 
-  @redhat-sso-7 @jboss-datavirt-6 @jboss-datagrid-6 @jboss-datagrid-7 @jboss-processserver-6 @jboss-decisionserver-6 
+  @redhat-sso-7 @jboss-datavirt-6 @jboss @jboss @jboss-processserver-6 @jboss-decisionserver-6
```
Comment (jmtd) [inline tests/features/eap/eap_common.feature]: bad substitution

File: tests/features/eap/eap_gc.feature [hunk 4]
```diff
@@ -1,4 +1,4 @@
-@jboss-decisionserver-6 @jboss-processserver-6 @jboss-datagrid-6 @jboss-datagrid-7 @jboss-datavirt-6 
+@jboss-decisionserver-6 @jboss-processserver-6 @jboss @jboss @jboss-datavirt-6
```
Comment (jmtd) [inline tests/features/eap/eap_gc.feature]: bad substitution

**Pull request merged**